scripts/validation_core.py

```diff
@@ -0,0 +1,60 @@
+
+from genologics.lims import *
+from genologics.config import BASEURI, USERNAME, PASSWORD
+from scilifelab.db.statusDB_utils import load_couch_server
+
+import LIMS2DB.objectsDB.objectsDB as newDB
+import scilifelab.lims_utils.objectsDB as oldDB
+import datetime
+import argparse
+
+def validate_cores(args):
+    mylims = Lims(BASEURI, USERNAME, PASSWORD)
+    pjs=mylims.get_projects()
+    couch = load_couch_server(args.conf)
+    proj_db = couch['projects']
+    samp_db = couch['samples']
+    tested=0
+    passed=0
+    failed=[]
+    for p in pjs:
+        if p.open_date:
+            opDate=datetime.datetime.strptime(p.open_date, '%Y-%m-%d')
+            now=datetime.datetime.now()
+
+            if now-opDate < datetime.timedelta(days=args.days):
+                tested+=1
+                print "comparing project {0}".format(p.name)
+                oldpj= oldDB.ProjectDB(mylims, p.id, samp_db)
+                newpj= newDB.ProjectDB(mylims, p.id, samp_db)
+                if oldpj.obj == newpj.obj:
+                    print "passed"
```

Comment:
Fair enough :-)

However, and this is a question: Do you put the output of this validation in some kind of official validation document? In that case would make more sense to use logging, as you will have date and time in the messages.